runtimes/CMakeLists.txt

```diff
@@ -155,6 +155,9 @@ set(LLVM_COMPILER_CHECKED ON)
 include(AddLLVM)
 include(HandleLLVMOptions)
 
+# Loot at the PATH first to avoid a version mismatch between the command-line
+# python and the CMake-found version
+set(Python3_FIND_REGISTRY LAST)
```

Comment:
FYI - ideally we wouldn't need to have things like this in the main CMakeLists.txt.

Ideally we'd set it in the CI environment, as we happen to need it there due to how the Github runners are set up. It is possible to pass this on the CMake command line, I think, like `-DPython3_FIND_REGISTRY=LAST`, but the `run-buildbot` script doesn't support passing extra command line parameters to cmake.

On the other hand - this doesn't really hurt either; the cmake default behaviour is really really surprising here, and this makes it less surprising. It makes things a little inconsistent to keep setting this in the `runtimes/CMakeLists.txt` only - I wonder if we could set it e.g. `cmake/Modules/CMakePolicy.cmake` as a sort of project-wide default policy?

Anyway, just venting my thoughts on the matter - no urgent action needed.